.evergreen/.setup_env

```diff
@@ -1,9 +1,15 @@
 set -e
 set -x
 export BASEDIR="$PWD/.evergreen"
-export PATH="$BASEDIR/node-v$NODE_JS_VERSION-win-x64:/opt/python/3.6/bin:/opt/chefdk/gitbin:/cygdrive/c/Program Files/Git/bin:/cygdrive/c/Program Files/Git/mingw32/libexec/git-core:/cygdrive/c/Python39/Scripts:/cygdrive/c/Python39:/cygdrive/c/cmake/bin:$PATH"
+export PATH="$BASEDIR/node-v$NODE_JS_VERSION-win-x64:/opt/chefdk/gitbin:/cygdrive/c/Program Files/Git/bin:/cygdrive/c/Program Files/Git/mingw32/libexec/git-core:/cygdrive/c/cmake/bin:$PATH"
 export IS_MONGOSH_EVERGREEN_CI=1
 
+if [ "$USE_PYTHON2" != "true" ]; then
+  export PATH="/opt/python/3.6/bin:/cygdrive/c/Python39/Scripts:/cygdrive/c/Python39:$PATH"
+else
+  echo "Using Python 2 - if you try to run tests these will most likely fail!"
+fi
+
```

Comment:
Can we do something similar to https://github.com/mongodb-js/mongosh/pull/833/files#diff-2f08d8a15f4dac5750b97ecd68c88c25565a273b1a6c7cab867215c3edf657c0R16 and explicitly set the PATH to one where python 2.x is at the front?

Reply:
Yeah sure, I can just make that change in the `package_and_upload` script, too.